src/server/index.js

```diff
@@ -8,20 +8,17 @@ const heartbeat = require('./heartbeat.js');
 
 const PORT = process.env.PORT || 3000;
 const app = express();
-const webpack = require('webpack');
-const webpackConfig = require('../../webpack.config.js');
-const compiler = webpack(webpackConfig);
-
-app.use(require('webpack-dev-middleware')(compiler, {
-  noInfo: true, publicPath: webpackConfig.output.publicPath,
-}));
 
 app.use(express.static(path.resolve(__dirname, '../../dist')));
 
 app.get('/login', (req, res) => {
```

Comment:
`app.get` can have an array of endpoints as the first parameter

> A string representing a path.
> A path pattern.
> A regular expression pattern to match paths.
> An array of combinations of any of the above.

http://expressjs.com/en/api.html#app.get.method